src/functional/resources/sample_full.csv

```diff
@@ -1,3 +1,3 @@
 Organization,Repository,branch,Student1 Git ID,Student1 Display Name,Student1's Git Alias,Student2 Git ID,Student2 Display Name,Student2's Git Alias,Student3 Git ID,Student3 Display Name,Student3's Git Alias,Student4 Git ID,Student4 Display Name,Student4's Git Alias
-CS2103AUG2017-W09-B1,main,master,nbriannl,,,zacharytang,Zachary Tang,Zachary Tang,April0616,Fan Yuting,LAPTOP-7KFM2KSP\User;Fan Yuting,CindyTsai1,Cindy Tsai,YuHsuan
-CS2103AUG2017-W09-B2,main,master,charlesgoh,,Charles Goh,Esilocke,,,jeffreygohkw,,Jeffrey Goh,wangyiming1019,,ACER\kyle;Wang Yiming
+reposense,testrepo-Beta,master,nbriannl,,,zacharytang,Zachary Tang,Zachary Tang,April0616,Fan Yuting,LAPTOP-7KFM2KSP\User;Fan Yuting,CindyTsai1,Cindy Tsai,YuHsuan
+reposense,testrepo-Charlie,master,charlesgoh,,Charles Goh,Esilocke,,,jeffreygohkw,,Jeffrey Goh,wangyiming1019,,ACER\kyle;Wang Yiming
```

Comment:
just a feedback from prof earlier - try not to set their name as display name in the report.